src/sentry/integrations/vsts/notify_action.py

```diff
@@ -18,10 +20,11 @@ class AzureDevopsCreateTicketAction(TicketEventAction):  # type: ignore
 
     def generate_footer(self, rule_url: str) -> str:
         return "\nThis work item was automatically created by Sentry via [{}]({})".format(
-            self.rule.label,
+            # TODO(mgaeta): Bug: Rule is optional.
```

Comment:
Is this a typing bug or an actual bug? Is this fixed somewhere?

Reply:
Yeah this is a real bug, it'll take some refactoring to handle the option.